src/me/StevenLawson/TotalFreedomMod/TFM_Admin.java

```diff
@@ -46,6 +48,7 @@ public TFM_Admin(UUID uuid, ConfigurationSection section)
         this.isSeniorAdmin = section.getBoolean("is_senior_admin", false);
         this.isTelnetAdmin = section.getBoolean("is_telnet_admin", false);
         this.consoleAliases = section.getStringList("console_aliases");
+        this.cmdSpyEnabled = section.getBoolean("cmdspy_is_on", false);
```

Comment:
The reason I made it false it to prevent everyone from getting commandspy when they first join if the admin list is wiped.